src/components/forms/AppSearchInput.vue

```diff
@@ -4,7 +4,24 @@
     @submit.prevent="emit('update:modelValue', searchText)"
   >
     <AppInput v-model="searchText" class="pr-8" :placeholder="placeholder" />
```

Comment:
I'm guessing pr-8 needs to update to pr-16 if the clear button is present, so the input text doesn't overflow into the button